environment-dev.yml

```diff
@@ -1,4 +1,4 @@
-name: pyturbo
+name: pyturbo-demos
```

Comment:
no need for this add ?

Reply:
I see, there is another file. maybe suffix `-dev` like the filename ?

Reply:
I changed it